taxi/forms.py

```diff
@@ -0,0 +1,30 @@
+from django import forms
+from django.core.validators import RegexValidator
+
+from taxi.models import Driver
+
+
+class DriverCreateForm(forms.ModelForm):
+    password = forms.CharField(widget=forms.PasswordInput())
+    license_number = forms.CharField(required=True,
+                                     min_length=8,
+                                     max_length=8,
+                                     validators=[
+                                         RegexValidator("^[A-Z]{3}\d{5}")])
```

Comment:
better create `clean_license_number` function, check the license and raise ValidationError with a message for each problem

Reply:
I added help_text label especially for user can understand which format he needs to input. And now my form look like:
<img width="805" alt="image" src="https://user-images.githubusercontent.com/60425345/178783679-cb2c1588-dc32-45bf-a6ba-a5a783f836db.png">
This view is more standardized I think. What about you think?

Reply:
With help_text looks ok but make it more descriptive. The example doesn't help users to understand what rules they must follow.

Reply:
With help_text looks ok but make it more descriptive. The example doesn't help users to understand what rules they must follow.

Reply:
I think a custom method is better because you can give a user feedback for each single mistake when they try to enter license_number. It can help users to spend less time filling the form. But variant with descriptive help_text is also ok

Reply:
<img width="530" alt="image" src="https://user-images.githubusercontent.com/60425345/178950888-bf8cfc8e-7d9a-4d16-a4b4-2ea20d8b9e4b.png">
You're right, now it looks really more descriptive.